exc_cell_splitting/make_new_inhib_exc_outside_SSCX.py

```diff
@@ -15,95 +14,75 @@
 #         ME type excitatory nrrd files
 ###############################################################################
 
-#packages
+import logging
 import os
-import re
+
+from pathlib import Path
+
 from voxcell import VoxelData
 from voxcell.nexus.voxelbrain import Atlas
-import numpy as np
 
-#load an nrrd
-def find_regions(fn_cand, root, format="path"):
-    #for fn_cand in fn_regions:
-    if 1:
-        expected_fn_regions = os.path.join(root, fn_cand)
-        if os.path.isfile(expected_fn_regions):
-            if format == "path":
-                return expected_fn_regions
-            elif format == "voxcell":
-                return VoxelData.load_nrrd(expected_fn_regions)
-            raise ValueError("Unknown format spec: {0}".format(format))
-    return None
+import numpy as np
+import pandas as pd
 
 
-#make excitatory density
-neuron_nrrd= find_regions("neuron_density_v3.nrrd",'./', "voxcell")
-inh_nrrd= find_regions("gad67+_density_v3.nrrd",'./', "voxcell")
-exc_nrrd=neuron_nrrd
-exc_nrrd.raw=neuron_nrrd.raw-inh_nrrd.raw
-exc_nrrd.raw=exc_nrrd.raw*(exc_nrrd.raw>0)
+logging.basicConfig(level=logging.INFO)
 
-#Do remapping
+L = logging.getLogger(__name__)
 
-#read in mapping file
-import csv
-allrows=[]
-with open('mapping_cortex_all_to_exc_mtypes.csv', newline='') as csvfile:
-    reader = csv.reader(csvfile, delimiter=',', quotechar='|')
-    for row in reader:
-        allrows.append(row)
-        #print(', '.join(row))
 
+#make excitatory density
+neuron_density = VoxelData.load_nrrd("neuron_density_v3.nrrd")
+inh_density = VoxelData.load_nrrd("gad67+_density_v3.nrrd")
+exc_density = neuron_density.with_data(np.clip(neuron_density.raw - inh_density.raw, a_min=0, a_max=None))
 
-#load atlas
+excitatory_mapping = pd.read_csv('mapping_cortex_all_to_exc_mtypes.csv').set_index('layer')
 atlas = Atlas.open('.')
+region_map = atlas.load_region_map()
 brain_regions = atlas.load_data('brain_regions')
 
+#layer names, starting with layer 2 and going to layer 6
+layer_names = {
+    'L1': ['SS1', 'SSp-bfd1', 'SSp-ll1', 'SSp-m1', 'SSp-n1', 'SSp-tr1', 'SSp-ul1', 'SSp-un1', 'SSp1', 'SSs1', 'VISrll1'],
```

Comment:
@dkeller9 can you check these values? I grabbed them from the ids you had before, and I"m confused why L2, for instance, doesn't have `SSp-bfd2` (id: 614454292)

Reply:
I'm also confused that the SSCX is targeted, when the rest of the repository deals with the Isocortex; for instance, there is a method to get the ids related to the isocortex: https://github.com/BlueBrain/atlas-densities/blob/00efaa1b7fcc4e06b96c490fcd30c66ef0626e9a/atlas_densities/densities/utils.py#L322

Reply:
@mgeplf that is a good point. I thought we were building SSCX and generic types outside. Should we be building cortex with SSCX types and generic outside?

Reply:
@dkeller9 I'm not sure I can answer that, unfortunately.

Reply:
The ME-type mapping of the inhibitory neuron molecular types was done by Yann for the whole isocortex region.

Reply:
Okay. I will modify this code so that it assigns the excitatory SSCX types to all of isocortex and not just SSCX. @mgeplf  do you have any suggestions on how to pull out individual layers of all of isocortex?